docs/faq.md

```diff
@@ -557,6 +557,35 @@ $ docker manifest inspect openjdk:8
 }
 ```
 
+You can inspect the digest (`@sha256:...`) of a manifest image with the `--verbose` option.
+```
+$ docker manifest inspect --verbose openjdk:8
+[
+	{
+		"Ref": "docker.io/library/openjdk:8@sha256:74a0711547137cd3787c0b84266a7ba4cb814b98b8950cf95bc2e526bfc05e47",
+    ...
+  }
+]    
+   ...
+   // This confirms that openjdk:8 points to a manifest list.
+   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
+   "manifests": [
+      {
+         // This entry in the list points to the manifest for the ARM64/Linux manifest.
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         ...
+         "digest": "sha256:1fbd49e3fc5e53154fa93cad15f211112d899a6b0c5dc1e8661d6eb6c18b30a6",
+         "platform": {
+            "architecture": "arm64",
+            "os": "linux",
+            "variant": "v8"
+         }
+      }
+   ]
+}
```

Comment:
Perhaps we can remove this section since we've already mentioned this above.

Reply:
Good suggestion thanks @mpeddada1 .  I've updated to make relevance of output clearer  (eee6882), and removed duplication.